### PR TITLE
Added logic to provide a count of total members

### PR DESCRIPTION
**Pull request by minMaximilian**
hopefully should work 🤞

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -240,3 +244,34 @@ func getAnnouncements(w http.ResponseWriter, r *http.Request) {
...
+		http.Error(w, "Please provide an int as 'q's value", 403)
+		return
+	}
+	if amount > limit {
+		http.Error(w, "Query amount exceeds the query limit", 403)
+		return
+	}
+
+	members, err := session.GuildMembers(servers.PublicServer, "", 1000)
+	if err != nil {
```
Comment (OisinA) [inline api/api.go:268]: Should probably return some sort of error code here

**Review by OisinA — CHANGES_REQUESTED** [review]

File: api/api.go [hunk 2]
```diff
@@ -240,3 +244,34 @@ func getAnnouncements(w http.ResponseWriter, r *http.Request) {
...
 
 }
+
+func getMembers(w http.ResponseWriter, r *http.Request) {
+	query := r.URL.Query()
+	limit := viper.GetInt("api.event_query_limit")
+	servers := viper.Get("discord.servers").(*config.Servers)
+	queryAmount, exists := query["q"]
+	if !exists || len(query) == 0 {
+		http.Error(w, "Please add the parameter 'q'", 403)
```
Comment (OisinA) [inline api/api.go:254]: 400 (bad request) is probably more suited than 403 code

File: api/api.go [hunk 3]
```diff
@@ -240,3 +244,34 @@ func getAnnouncements(w http.ResponseWriter, r *http.Request) {
...
+		return
+	}
+	amount, err := strconv.Atoi(queryAmount[0])
+	if err != nil {
+		http.Error(w, "Please provide an int as 'q's value", 403)
```
Comment (OisinA) [inline api/api.go:259]: Same here

File: api/api.go [hunk 4]
```diff
@@ -240,3 +244,34 @@ func getAnnouncements(w http.ResponseWriter, r *http.Request) {
...
+		return
+	}
+	if amount > limit {
+		http.Error(w, "Query amount exceeds the query limit", 403)
```
Comment (OisinA) [inline api/api.go:263]: And here

**Review by OisinA — CHANGES_REQUESTED** [review]

File: api/api.go (hunk 2 above)
Comment (minMaximilian) [reply api/api.go:254]: I was just following the codes as stated in previous API requests, I can always just send another PR request to fix all the codes

Comment (OisinA) [reply api/api.go:254]: Depends what @ericm says, not sure why they'd all be 403

File: api/api.go (hunk 4 above)
Comment (Thomas007G) [reply api/api.go:263]: status 400 for bad request 403 is access denied

Comment (ocanty) [reply api/api.go:263]: 500 bruh moment

**Comment by minMaximilian** [comment]
RE: Previous
Figured some code was unnecessary, added http errors with proper status codes 👀

**Review by ericm — APPROVED** [review]

**Review by OisinA — APPROVED** [review]

**Pull request merged**